### ISSUE TITLE
Only route specific traffic via tunnel

### ISSUE THREAD

**Issue by besselfunct** [fabi321/icmptunnel-rs#1]
Hi,

I came across this project, and it looks really great. If I'm reading the `configure_network.rs` file correctly, it looks like this is creating a new virtual network interface called `tun0` and then routing all(?) traffic through that? Is there a way to specify on the client side that I only want to route traffic for a given port (say 0.0.0.0:SomePort) through the icmp tunnel?

Thanks!

**Comment by fabi321** [comment]
Yes, you do read that right, it does route all traffic through the new `tun0` interface.

If you only want to route specific ports, you'll have to manually configure routes. If you want to, I could add a cli option to turn the automatic setup off, and you'd need to manually configure rules using ipconfig or whatever else you prefer.
You can also manually add the gateway back in right now, and configure the tun0 interface only for specific traffic.

I've never heard of routing by port, but routing by IP-Address should be doable.
